src/components/AdminForm.js

```diff
@@ -43,4 +43,18 @@ const AdminForm = () => {
   );
 }
 
-export default AdminForm;
\ No newline at end of file
+const mapStateToProps = state => {
+  return {
+    cohortName: state.cohortInfo.adminFormReducer,
```

Comment:
adminFormReducer should be replaced with cohortName and cohortType otherwise state is not getting updated

Reply:
Fixed this, good catch!